ras_party/views/enrolments_view.py

```diff
@@ -58,4 +61,17 @@ def get_respondent_enrolments(party_uuid: UUID) -> Response:
         )
         return BadRequest()
 
-    return make_response([enrolment.to_dict() for enrolment in enrolments], 200)
+    return make_response(enrolments, 200)
+
+
+@enrolments_view.route(
+    "/is_respondent_enrolled/<party_uuid>/business_id/<business_id>/survey_id/<survey_id>", methods=["GET"]
+)
+def get_is_respondent_enrolled(party_uuid: UUID, business_id: UUID, survey_id: UUID) -> Response:
```

Comment:
There is similar endpoint called validate_respondent_claim, I am not a fan and will look to use just this one in the future. The only reason I haven't done it now is that is used by secure message, and rather than expanding the scope and having to update a 3rd repo, I would rather do a small change there after this PR is on